packages/server/graphql/private/typeDefs/_legacy.graphql

```diff
@@ -1049,6 +1049,397 @@ interface NewMeeting {
   locked: Boolean!
 }
 
+"""
+The user account profile
+"""
+type User {
```

Comment:
-1 We don't need to repeat public types in the private schema anymore. This only leads to code duplication. I will do a cleanup of all these duplicates soon, in the meantime please don't add any back.